diffstat for openjade-1.4devel1 openjade-1.4devel1

 changelog |    9 +++++++++
 rules     |    1 +
 2 files changed, 10 insertions(+)


No differences were encountered between the control files

diff -Nru openjade-1.4devel1/debian/changelog openjade-1.4devel1/debian/changelog
--- openjade-1.4devel1/debian/changelog	2016-10-01 17:30:14.000000000 +0200
+++ openjade-1.4devel1/debian/changelog	2016-10-01 17:30:15.000000000 +0200
@@ -1,3 +1,12 @@
+openjade (1.4devel1-21.3) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Pass -fno-lifetime-dse to GCC to work around invalid assumptions about
+    object lifetimes (closes: #823419). Thanks to Daniel Schepler for the bug
+    report.
+
+ -- Jakub Wilk <jwilk@debian.org>  Sat, 01 Oct 2016 17:01:25 +0200
+
 openjade (1.4devel1-21.2) unstable; urgency=medium
 
   * Non-maintainer upload.
diff -Nru openjade-1.4devel1/debian/rules openjade-1.4devel1/debian/rules
--- openjade-1.4devel1/debian/rules	2016-10-01 17:30:14.000000000 +0200
+++ openjade-1.4devel1/debian/rules	2016-10-01 17:30:15.000000000 +0200
@@ -31,6 +31,7 @@
   CFLAGS += -ffunction-sections
   CXXFLAGS += -ffunction-sections
 endif
+CXXFLAGS += -fno-lifetime-dse  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=69534
 export CFLAGS
 export CXXFLAGS
 
